server.js

```diff
@@ -19,4 +20,6 @@ app.post('/api/foods', FoodsController.create);
 app.patch('/api/foods/:id', FoodsController.update);
 app.delete('/api/foods/:id', FoodsController.destroy);
 
+app.get('/api/meals/:category/:year/:month/:day', MealsController.index);
```

Comment:
But the actual route as defined in the code would be /meals/:category and the request would pass date through in params (I think?).